CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Misc:
 
 Fixes:
 - [#1814](https://github.com/rails-api/active_model_serializers/pull/1814) Ensuring read_multi works with fragment cache
+- [#1848](https://github.com/rails-api/active_model_serializers/pull/1848) Redefine associations on inherited serializers
```

Comment:
can you add `(@EhsanYousefi )` to the end of this? :-) gotta make sure you get credit!